### DISCUSSION TITLE
XIAO RP2040/RP2350 WITH WIFI MODULE

### DISCUSSION THREAD

**Discussion by usermctech** [Seeed-Studio/OSHW-XIAO-Series#24] [Wish List]
XIAO RP2040/RP2350 WITH WIFI MODULE

**Comment by meilily** [comment]
Hi @usermctech, we didn't include a Wi-Fi module for the XIAO RP2040 and RP2350 because we couldn't fit in any extra module to the extreme size of 17.8x21mm (think of 2 gummy bears). However, if you'd love to have Wi-Fi connectivity for XIAO, you can consider using our XIAO ESP32 Boards (XIAO ESP32C3, XIAO ESP32C6, XIAO ESP32S3, and XIAO ESP32S3 Sense). :)
